Source/Paths/SvgPathBuilder.cs

```diff
@@ -217,31 +224,38 @@ private static PointF ToAbsolute(float x, float y, SvgPathSegmentList segments,
             return point;
         }
 
-        private static IEnumerable<string> SplitCommands(string path)
+        private static List<(int Start, int Length)> SplitCommands(ref ReadOnlySpan<char> path)

```

Comment:
Is there a reason why the return type is not List<(ReadOnlySpan<char>) where the slices are returned

Reply:
You can use ReadOnlySpan<char> as type argument.

Reply:
The best solution is to inline the SplitCommands so no List or enumerator allocations are made.

Reply:
Or the return type could be an enumerable and the slices could be yielded.

Reply:
Not possible, can not return slices.

Reply:
I will just inline SplitCommands as it waste too much resources by allocating list or enumerable.